components/layout_2020/display_list/stacking_context.rs

```diff
@@ -1404,17 +1404,31 @@ impl BoxFragment {
 
         // Non-scrollable overflow path
         if overflow.x == ComputedOverflow::Clip || overflow.y == ComputedOverflow::Clip {
+            // TODO: update this to the proper box after the parser is ready
```

Comment:
Do you mind clarifying this comment while you do this cleanup? What parser?

Reply:
Changed to

```rs
            // TODO: The spec allows `overflow-clip-rect` to specify which box edge to use
            // as the overflow clip edge origin, but Stylo doesn't currently support that.
            // It will need to be handled here, for now always use the padding rect.
```